projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/ui-handler/with-ui-handler-data-provider.tsx

```diff
@@ -1,8 +1,10 @@
 /**
  * External dependencies
  */
+import { parse } from '@wordpress/blocks';
```

Comment:
```suggestion
import { useAiContext } from '@automattic/jetpack-ai-client';
import { parse } from '@wordpress/blocks';
```
because of [this](https://github.com/Automattic/jetpack/pull/32209/files#r1281439708).